explorer/src/lib/dusk/components/nav-list/styles.css

```diff
@@ -0,0 +1,48 @@
+.dusk-nav-list {
+  display: flex;
+  flex-direction: column;
+  row-gap: 3.5rem;
+  height: 100%;
+}
+
+.dusk-nav-list__link {
+  display: block;
+  text-transform: uppercase;
+  font-size: 1.5rem;
+  font-weight: 500;
+  line-height: 1;
+  text-decoration: none;
+  color: var(--smokey-block);
```

Comment:
In theory components shouldn't use named colours, but rather "conceptual" colours.
e.g. if you have a `--background-color` as a background color, you should use its `--on-*` variant (`--on-background-color`).

Same applies to the colours below in this CSS.